### PR TITLE
Add 2021 Accord 2.0 Touring & Hybrid Touring

### PR DESCRIPTION
**Pull request by csouers**
- Create the HONDA_BOSCH_EXT set for HONDA_BOSCH cars that utilize the new extended IDs for LKAS_HUD
- Create a new CAR for each 2021 variant. Let's us differentiate between which LKAS_HUD frames to send to the radar.
 
These cars are basically the same as prior models with the aforementioned changes to LKAS_HUD on the private CAN.

**Pre reqs**
- opendbc > [pr](https://github.com/commaai/opendbc/pull/372)
- panda > [pr](https://github.com/commaai/panda/pull/615)
- add `085ac1d942c35910|2021-03-25--20-11-15` to the test bucket

**Checklist**
- [x] added to README
- [x] test routes added to [test_car_models](../../selfdrive/test/test_car_models.py)
- [x] routes with openpilot: [2.0 Touring](https://my.comma.ai/cabana/?route=085ac1d942c35910%7C2021-03-31--19-35-33&exp=1648783384&sig=aqbsNvoXEpgy4IjKdQ8g%2FZESG%2FQJP9uFQNQDGq%2Fx02I%3D&max=9&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F085ac1d942c35910%2Ff24803fb096e557ecae92aa832461cd4_2021-03-31--19-35-33) [Hybrid Touring](https://my.comma.ai/cabana/?route=f29e2b57a55e7ad5%7C2021-04-01--13-04-26&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Ff29e2b57a55e7ad5%2F3d99aebd0e6345d0a239b7a55c950dca_2021-04-01--13-04-26&seekTime=2)
- [x] routes with stock system: [2.0 Touring](https://my.comma.ai/cabana/?route=085ac1d942c35910%7C2021-03-25--20-11-15&exp=1648267404&sig=9hSlw2R5TqsNuiD4IEc9OJOhPuaVAGe9Zvmr9BUQ9KE%3D&max=21&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F085ac1d942c35910%2Fb02baf43008450d9bc6c4429e971e9f0_2021-03-25--20-11-15) [Hybrid Touring](https://my.comma.ai/cabana/?route=f29e2b57a55e7ad5%7C2021-03-24--20-52-38&exp=1648771523&sig=E%2Fl%2Fw19L0WFueauNE%2B%2BpYHL4l%2BxVbh97GEco0X6Hr%2B8%3D&max=19&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Ff29e2b57a55e7ad5%2F5db5e3a6633f68d79d03c775f0da097e_2021-03-24--20-52-38)

Special thanks to: kyled, driftingnumja, fancypantsless, kn, ctrlatldel, speedhacks, kaleo, ankitgu, and any others I may have missed.

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -63,12 +65,18 @@ class CAR:
   CAR.ACCORD: [{
     148: 8, 228: 5, 304: 8, 330: 8, 344: 8, 380: 8, 399: 7, 419: 8, 420: 8, 427: 3, 432: 7, 441: 5, 446: 3, 450: 8, 464: 8, 477: 8, 479: 8, 495: 8, 545: 6, 662: 4, 773: 7, 777: 8, 780: 8, 804: 8, 806: 8, 808: 8, 829: 5, 862: 8, 884: 8, 891: 8, 927: 8, 929: 8, 1302: 8, 1600: 5, 1601: 8, 1652: 8
   }],
+  CAR.ACCORD_2021: [{
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/values.py:68]: remove the can fingerprints, they're deprecated now

File: selfdrive/car/honda/values.py [hunk 2]
```diff
@@ -151,7 +159,7 @@ class CAR:
 }
 
 # Don't use theses fingerprints for fingerprinting, they are still needed for ECU detection
-IGNORED_FINGERPRINTS = [CAR.INSIGHT, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_EU, CAR.HRV, CAR.ACURA_RDX_3G]
+IGNORED_FINGERPRINTS = [CAR.INSIGHT, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_EU, CAR.HRV, CAR.ACURA_RDX_3G, CAR.ACCORD_2021, CAR.ACCORDH_2021]
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/values.py:162]: rebase on master. this is gone

File: selfdrive/car/honda/values.py [hunk 3]
```diff
@@ -335,21 +343,17 @@ class CAR:
   CAR.ACCORDH: {
     (Ecu.gateway, 0x18daeff1, None): [
       b'38897-TWA-A120\x00\x00',
-      b'38897-TWD-J020\x00\x00',
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/values.py:338]: why are these removed?

Comment (csouers) [reply selfdrive/car/honda/values.py:318]: This commit fixes an ID conflict. These were added with https://github.com/commaai/openpilot/pull/20850 and they match a possible 2021 Hybrid. Some of the deleted IDs could be from the older models, but I have no way of knowing which cars they're really from. @pd0wm are you able to see if these came from a car that doesn't have `36161-TWA-A330\x00\x00` or `36161-TVC-A330\x00\x00`?

Removing at least the camera and radar IDs (definitely only for 2021+) could fix the conflict.

Comment (csouers) [reply selfdrive/car/honda/values.py]: I've rebased and limited commit #2 to just the camera and radar firmware ids for the 2021 Hybrid. I'm happy to submit the PR to clean up the rest later once you all are ready for that.

**Comment by feyded1020** [comment]
#21393 Made a new PR, and closed the old one that adeebshihadeh mentioned.

**Comment by adeebshihadeh** [comment]
Can you rebase on master? I merged the opendbc PR and bumped the submodule on master. The 2021 Accords should use the 2018 Accord DBC.

**Comment by csouers** [comment]
Rebased this and applied the needed changes. Working on the panda PR now.

File: selfdrive/car/honda/values.py [hunk 4]
```diff
@@ -1306,5 +1384,6 @@ class CAR:
   CAR.INSIGHT: 1.,
 }
 
-HONDA_BOSCH = set([CAR.ACCORD, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_5G, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G])
-HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
+HONDA_BOSCH = set([CAR.ACCORD, CAR.ACCORD_2021, CAR.ACCORDH, CAR.ACCORDH_2021, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_5G, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G])
+HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.ACCORD_2021, CAR.CRV_5G, CAR.ACURA_RDX_3G])
+HONDA_BOSCH_EXT = set([CAR.ACCORD_2021, CAR.ACCORDH_2021])
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/values.py:1389]:
```suggestion

# Bosch models with alternate set of LKAS_HUD messages
HONDA_BOSCH_EXT = set([CAR.ACCORD_2021, CAR.ACCORDH_2021])
```

**Comment by adeebshihadeh** [comment]
Fixed the conflicts in #22007 since I wasn't able to push to your branch for some reason.
